docs/Preferences.md

```diff
@@ -65,7 +65,7 @@
   ]
   ```
 
-  Please see [XSD file associations](Validation.md#xml-file-association-with-xsd) and [DTD file associations](Validation.md#xml-file-association-with-dtd) for more information.
+Please see [XSD file associations](Validation.md#xml-file-association-with-xsd)and [DTD file associations](Validation.md#xml-file-association-with-dtd) for more information.
```

Comment:
That space before "and" was actually needed :-)

Reply:
Good catch! fixed.